assets/kubernetes/dashboards/statefulset.json

```diff
@@ -72,7 +72,7 @@
                "tableColumn": "",
                "targets": [
                   {
-                     "expr": "sum(rate(container_cpu_usage_seconds_total{job=\"cadvisor\", cluster=\"$cluster\", namespace=\"$namespace\", pod=~\"$statefulset.*\"}[3m]))",
+                     "expr": "sum(rate(container_cpu_usage_seconds_total{job=\"cadvisor\", cluster=\"$cluster\", container!=\"\", namespace=\"$namespace\", pod=~\"$statefulset.*\"}[3m]))",
```

Comment:
Looks like `make` pulled in some daily changes. I can remove these from PR if desired.